src/components/productCardShowMoreButton.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Image from "next/image";
+
+type Content = {
+  imageAlt: string;
+  image: string[];
+};
```

Comment:
Potentially have to work on naming convention for your types. There is already 3 types called Content and they're all different

Reply:
updated naming to be more descriptive